PySDM/backends/impl_numba/methods/seeding_methods.py

```diff
@@ -0,0 +1,57 @@
+""" CPU implementation of backend methods for particle injections """
+
+from functools import cached_property
+
+import numba
+
+from PySDM.backends.impl_common.backend_methods import BackendMethods
+
+
+class SeedingMethods(BackendMethods):  # pylint: disable=too-few-public-methods
+    @cached_property
+    def _seeding(self):
+        @numba.njit(**{**self.default_jit_flags, "parallel": False})
+        def body(
+            idx,
+            multiplicity,
+            extensive_attributes,
+            seeded_particle_multiplicity,
+            seeded_particle_extensive_attributes,
+            number_of_super_particles_to_inject: int,
+        ):
+            # TODO #1367 it should be possible to start enumerating from the end of valid particle set
+            for i, mult in enumerate(multiplicity):
+                if number_of_super_particles_to_inject == 0:
+                    break
+                if mult == 0:
+                    idx[i] = -1
+                    multiplicity[i] = seeded_particle_multiplicity
+                    for a in range(len(extensive_attributes)):
+                        extensive_attributes[a, i] = (
+                            seeded_particle_extensive_attributes[a]
+                        )
+                    number_of_super_particles_to_inject -= 1
```

Comment:
This looks great!

It seems straight forward to me to extend this code structure so that the multiplicity and extensive attributes of each particle are stochastically sampled from some prescribed distribution rather than specified directly. Does that seem correct? If not, I'd say we can also hold off on adding that functionality for a second PR.

Reply:
We need to specify all non-dependent attributes, i.e. kappa, dry size, wet size, location in space, etc. Some of them are not independent, e.g., dry vs. wet size. In this case, the environment RH might actually be also needed to specify the wet size in a way that the condensation solver does not do crazy things right after seeding (and the needed RH is dependent on x,y,z).  But, if we assume that seeding is only done inside cloud, probably it doesn't matter that much?

Reply:
see https://github.com/open-atmos/PySDM/pull/1367/commits/c17f338ad2ff5a6b4a8b62f9033b75e8f43aa6d8 for a first draft of a sampling logic